digital_image_processing/rotation/rotation.py

```diff
@@ -0,0 +1,50 @@
+from matplotlib import pyplot as plt
+from numpy import float32
+import cv2
+
+
+def get_rotation(img, pt1, pt2, rows, cols):
```

Comment:
Type hints?  https://docs.python.org/3/library/typing.html

Reply:
Added type hints in get_rotation(). Thanks for the help!